addons/web/static/src/js/core/rpc.js

```diff
@@ -2,6 +2,7 @@ odoo.define('web.rpc', function (require) {
 "use strict";
 
 var ajax = require('web.ajax');
+var session = require('web.session');
```

Comment:
I would prefer not to add this dependency. Can't you add the companies info in session.rpc directly?